.mvn/settings.xml

```diff
@@ -1,16 +1,16 @@
-<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
-          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <servers>
         <server>
-            <id>bintray-corfudb-maven</id>
-            <username>corfudb</username>
-            <password>${secrets.jfrog_oss_password}</password>
+            <username>xchang@vmware.com</username>
```

Comment:
please use jfrog_oss_password as it was in the previous version, please just update user/password in github

Reply:
I mean we don't need to share outside, so far. If it would be needed we can do that in the future

Reply:
Yeah I will request for putting this password in our repo's secrets, and this password is already encrypted by the JFrog though, it's not the real password :)